tests/test_utils.py

```diff
@@ -0,0 +1,86 @@
+from cashews.utils import Bitarray, get_hashes
+
+
+def test_bitarray_get_size_1():
+    b = Bitarray("100010", 2)
+    assert b.get(0) == 0
+    assert b.get(1) == 1
+    assert b.get(2) == 0
+    assert b.get(3) == 0
+    assert b.get(4) == 0
+    assert b.get(5) == 1
+    assert b.get(6) == 0
+    assert b.get(7) == 0
+
+
+def test_bitarray_get_size_2():
+    b = Bitarray("100010", 2)
+    assert b.get(0, 2) == 2
```

Comment:
it should equal to `1`, since we sliced from the end and took only two bits (in that case - `0` and `1`)